infra/secrets.yaml

```diff
@@ -0,0 +1,29 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: ''
+Resources:
+  password:
+    Type: 'AWS::SecretsManager::Secret'
+    Properties:
+      Name: databasepassword
+      Description: "This is the password for the database, shared with the container."
+      GenerateSecretString:
+        PasswordLength: 40
+        ExcludeCharacters: '"@/\'
+  secretkey:
+    Type: 'AWS::SecretsManager::Secret'
+    Properties:
+      Name: secretkey
+      Description: "This is the secret key for the application."
+      GenerateSecretString:
+        PasswordLength: 40
+Outputs:
+  Key:
+    Description: The secret key for the application
+    Value: !Ref secretkey
+    Export:
+      Name: !Sub '${AWS::StackName}-key'
```

Comment:
**Question** (no change required): Does this resolve to the stack name of the whole thing `uat`? or does it resolve to the name of this file `Secrets`?

Reply:
I think it's the autogenerated stack name which starts with `Secrets`, based on people complaining about that online, but the docs don't say and it's not in the console, weird.

Reply:
Cool. Yeah, I am just curious. Not super important.